src/lang/queryAst.ts

```diff
@@ -105,9 +106,11 @@ export function getNodeFromPath<T>(
           .filter((a) => a)
           .join(' > ')}`
       )
-      console.error(tree)
-      console.error(sourceCode)
-      console.error(error.stack)
+      if (!suppressNoise) {
+        console.error(tree)
+        console.error(sourceCode)
+        console.error(error.stack)
+      }
```

Comment:
So there are cases where I expect a pathToNode to not be valid, and so it returning an error is useful signal, and not a case where I want to flood the console with noise.